CHANGELOG.md

```diff
@@ -15,6 +15,8 @@ WYMeditor.
 * [#764](https://github.com/wymeditor/wymeditor/pull/764)
   TablePlugin row/column remove:
   refrain from error when no selectedContainer
+* [#749](Register any modification after selection change)
```

Comment:
How about:

If a user modifies their selection via arrow keys or clicking, a new undo/redo savepoint is now automatically created. Previously, typing, moving selection, typing some more and then clicking undo would remove both typing actions. Now, only the second bit of typing will be undone.